doc/getting-started/rt_industry.rst

```diff
@@ -198,13 +198,17 @@ Use the ACRN industry out-of-the-box image
 
    ::
 
-      # efibootmgr -c -l "\EFI\acrn\acrn.efi" -d /dev/sda -p 1 -L "ACRN" -u "uart=disabled"
+      # efibootmgr -c -l "\EFI\acrn\acrn.efi" -d /dev/sda -p 1 -L "ACRN" -u "uart=disabled "
 
    Or use the following command to enable the serial port:
 
    ::
 
-      # efibootmgr -c -l "\EFI\acrn\acrn.efi" -d /dev/sda -p 1 -L "ACRN" -u "uart=port@0x3f8"
+      # efibootmgr -c -l "\EFI\acrn\acrn.efi" -d /dev/sda -p 1 -L "ACRN" -u "uart=port@0x3f8 "
+
+   .. note:: This is a workaround for the issue in efi-stub;
```

Comment:
Please add a reference to https://github.com/projectacrn/acrn-hypervisor/issues/4520. I guess this is the correct issue to be referenced, isn't it?